en/advanced_config/racer_setup.md

```diff
@@ -1,212 +1,200 @@
 # Racer Setup
 
-This page describes how to setup a racer and get the best performance out of it,
-particularly in acro mode.
+This page describes how to setup and configure a racer for optimal performance (in particular for [Acro mode](../flight_modes/acro_mc.md)).
 
-> **Tip** Many things described here can also be applied to other multicopters
-> to improve flight performance.
+Keep in mind that racers are fast vehicles, specifically designed to be overpowered! 
+You should already have some experience, or let someone with experience help you.
 
-Keep in mind that racers are fast vehicles, specifically designed to be
-overpowered, and thus you should already have some experience, or let someone
-with experience help you.
+> **Tip** Many things described here can also be applied to improve the flight performance of other types of multicopters.
 
-> **Note** A racer usually leaves away certain sensors, such as the GPS, which means that less failsafe options are available.
+<span></span>
+> **Note** A racer usually omits some sensors (e.g. GPS). As a result, fewer failsafe options are available.
 
 
 ## Build Options
-Racers typically do not have a GPS, but you will get some benefits if you add one.
-### Minimal
```

Comment:
I somewhat liked the clearer distinction between minimal vs. GPS, because PX4 has the reputation that it requires a mag and GPS in order to work. Not sure how to best highlight that though.